Training/Easy/horse_racing.rb

```diff
@@ -1,18 +1,18 @@
 # Solution for https://www.codingame.com/ide/puzzle/horse-racing-duals
 class HorseRacing
+  attr_accessor :horses_count
+
+  def initialize
+    @horses_count = gets.to_i
+  end
+
   def find_difference
```

Comment:
This method does two things - it inializes strengths and it also finds their difference. Try to refactor it so that it only does one thing